xml/update-alternatives.xml

```diff
@@ -0,0 +1,585 @@
+     <para>
+      You are changing any slave links too which are associated to a master
+      link. In case of our Java example, this will change the many slave
+      links you can see from <xref linkend="sec.ua.displaying"/>. Check
+      with <command>update-alternatives</command> <option>--display java</option>.
+     </para>
+    </listitem>
+   </itemizedlist>
+   <para>
+    If you change your mind, you can always switch back.
+   </para>
+  </sect1>
+
+  <sect1 xml:id="sec.ua.install">
+   <title>Installing your own Alternatives</title>
+   <para>
+    To create some alternatives in your system, we assume the following
+    properties:
+   </para>
+   <itemizedlist>
+    <listitem>
+     <para>
+      Two scripts, <command>foo-2</command> and <command>foo-3</command>,
+      which provides similar functionality.
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      The scripts are stored in the <filename>/usr/local/bin</filename>
+      directory to avoid any conflicts with the system tools in
+      <filename>/usr/bin</filename>.
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      A master link <command>foo</command> which points either to
+      <command>foo-2</command> or <command>foo-3</command>.
+     </para>
+    </listitem>
+   </itemizedlist>
+   <para>
+    In case you want to provide alternatives in your system, follow these
+    steps: 
+   </para>
+   <procedure>
+    <step>
+     <para>
+      Copy your scripts into the <filename>/usr/local/bin</filename> directory.
+     </para>
+    </step>
+    <step>
+     <para>
+      Make sure the scripts are executable:
+     </para>
+     <screen>&prompt.root;<command>chmod</command> +x /usr/local/bin/foo-{2,3}</screen>
+    </step>
+    <step>
+     <para>
+      Run <command>update-alternatives</command> for both scripts:
+     </para>
+     <screen>&prompt.root;<command>update-alternatives</command> --install \
+   /usr/local/bin/foo <co xml:id="co.ua.ua-install.usr-local-bin-foo"/>\
+   foo <co xml:id="co.ua.ua-install.foo"/>\
+   /usr/local/bin/foo-2 <co xml:id="co.ua.ua-install.usr-local-bin-foo.path"/>\
+   200 <co xml:id="co.ua.ua-install.prio"/>
+&prompt.root;<command>update-alternatives</command> --install \
+   /usr/local/bin/foo <xref linkend="co.ua.ua-install.usr-local-bin-foo"/>\
+   foo <xref linkend="co.ua.ua-install.foo"/>\
+   /usr/local/bin/foo-3 <xref linkend="co.ua.ua-install.usr-local-bin-foo.path"/>\
+   300 <xref linkend="co.ua.ua-install.prio"/></screen>
+     <para>
+      The options after <option>--install</option> means:
+     </para>
+     <calloutlist>
+      <callout arearefs="co.ua.ua-install.usr-local-bin-foo">
+       <para>
+        The generic name. To avoid confusion, this is usually the script
+        name without any version numbers.
+       </para>
+      </callout>
+      <callout arearefs="co.ua.ua-install.foo">
+       <para>
+        The name of the master link. This has to be the same for both scripts.
```

Comment:
*This has to be the same for both scripts* -> Must be the same

Reply:
done